change/@azure-msal-common-2020-09-21-10-44-34-appmetadata-foci.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "FOCI - Family of Client IDs feature",
```

Comment:
Nit: Are we putting pull request numbers in the comments?

Reply:
FOCI is a feature name. Do you mean we should add the PR number? Am I missing something?